package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "name": "ipr-api",
-  "version": "7.21.0",
+  "version": "7.22.0",
```

Comment:
shouldn't be incrementing the version in a pr to dev - or is dev not up to date with master?